lms/views/admin/organization.py

```diff
@@ -31,8 +32,13 @@ def show_organization(self):
     def update_organization(self):
         org = self._get_org_or_404(self.request.matchdict["public_id"])
 
-        org.name = self.request.params.get("name", "").strip() or None
+        self.organization_service.update_organization(
+            org,
+            name=self.request.params.get("name", "").strip(),
+            enabled=self.request.params.get("enabled", "") == "on",
+        )
 
+        AdminChangeEvent.notify_changes(self.request, org)
```

Comment:
`maybe_notify_changes`? I think the name is it's fine as it is. 

The method will not create the event if no changes are found.